JuiceMaker/JuiceMaker/Controller/ChangeStockViewController.swift

```diff
@@ -9,12 +9,81 @@ import UIKit
 
 final class ChangeStockViewController: UIViewController {
     
+    let currentFruitStock = FruitStore.shared.currentFruitStock
+    
+    lazy var fruitAndLabel: [Fruit: UILabel] = [
+        .strawberry: strawberryStockLabel,
+        .banana: bananaStockLabel,
+        .pineapple: pineappleStockLabel,
+        .kiwi: kiwiStockLabel,
+        .mango: mangoStockLabel
+    ]
+    
+    @IBOutlet weak var strawberryStockLabel: UILabel!
+    @IBOutlet weak var bananaStockLabel: UILabel!
+    @IBOutlet weak var pineappleStockLabel: UILabel!
+    @IBOutlet weak var kiwiStockLabel: UILabel!
+    @IBOutlet weak var mangoStockLabel: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        updateFruitStockOnLabel()
+        updateFruitStockOnStepper()
+    }
+    
+    private func updateFruitStockOnLabel() {
+        for (fruit, label) in fruitAndLabel {
+            guard let stock = currentFruitStock[fruit] else { return }
+            label.text = String(stock)
+        }
+    }
+    
+    private func updateFruitStockOnStepper() {
+        let fruitAndStepper: [Fruit: UIStepper] = [
+            .strawberry: strawberryStepper,
+            .banana: bananaStepper,
+            .pineapple: pineappleStepper,
+            .kiwi: kiwiStepper,
+            .mango: mangoStepper
+        ]
+        
+        for (fruit, stepper) in fruitAndStepper {
+            guard let stock = currentFruitStock[fruit] else { return }
+            stepper.value = Double(stock)
+        }
+    }
+    
+    @IBAction func changeStockOnLabel(by sender: UIStepper) {
+        let stepperAndLabel: [UIStepper: UILabel] = [
+            strawberryStepper: strawberryStockLabel,
+            bananaStepper: bananaStockLabel,
+            pineappleStepper: pineappleStockLabel,
+            kiwiStepper: kiwiStockLabel,
+            mangoStepper: mangoStockLabel
+        ]
+        
+        if let label = stepperAndLabel[sender] {
+            label.text = String(Int(sender.value))
+        }
+    }
+    
+    @IBAction func closeViewBarButton(_ sender: UIBarButtonItem) {
+        changedFruitStock()
+        dismiss(animated: true)
     }
     
-    @IBAction func changeConfirm(_ sender: Any) {
-        self.dismiss(animated: true)
+    private func changedFruitStock() {
+        for (fruit, label) in fruitAndLabel {
+            guard let stock = currentFruitStock[fruit] else { return }
+            guard let changedStock = label.text, let changedStock = Int(changedStock) else { return }
```

Comment:
둘 다 `changedStock`이라는 네이밍을 사용하고 있는데 둘은 타입이 다르므로 이를 반영한 네이밍을 해볼 수 있을 것 같습니다.

또한 `else` 블록 내부에 `return`을 제외한 코드가 없으므로 두 `guard` 문을 하나로 합쳐볼 수 있을 것 같습니다.
```suggestion
            guard let stock = currentFruitStock[fruit],
                  let changedStock = fruitComponent.label.text,
                  let changedStock = Int(changedStock) else { return }
```